modules/caddytls/connpolicy.go

```diff
@@ -409,8 +409,8 @@ func (clientauth ClientAuthentication) verifyPeerCertificate(rawCerts [][]byte,
 	return fmt.Errorf("client leaf certificate failed validation")
 }
 
-// decodeBase64DERCert base64-decodes, then DER-decodes, certStr.
-func decodeBase64DERCert(certStr string) (*x509.Certificate, error) {
+// DecodeBase64DERCert base64-decodes, then DER-decodes, certStr.
+func DecodeBase64DERCert(certStr string) (*x509.Certificate, error) {
```

Comment:
Since we don't need to double-verify the input, I don't think this needs to be exported anymore.